client/web/compose/src/components/PageBlocks/RecordEditor.vue

```diff
@@ -14,12 +14,13 @@
     <div
       v-else-if="module"
       class="mt-3 px-3"
+      :class="recordEditorMode"
     >
       <template v-for="field in fields">
         <div
           v-if="canDisplay(field)"
           :key="field.id"
-          class="field-container mb-3"
+          class="field-container mb-3 horizontal-mode"
```

Comment:
you need flex-fill and remove the mb-3 since you have gap now

Also instead of horizontal-mode class, just have a style="min-width: 13rem;"